code/Search/ContentControllerSearchExtension.php

```diff
@@ -52,7 +54,7 @@ public function SearchForm()
      * @param SearchForm $form The form instance that was submitted
      * @param HTTPRequest $request Request generated for this action
      */
-    public function results($data, $form, $request)
+    public function results(array $data, Form $form): HTTPResponse
```

Comment:
I don't think this is actually returning an `HTTPResponse`?
The fact that didn't cause CI to fail means we probably need a new test for this.

Reply:
Reverted

Reply:
Regarding this any any other comments saying "we probably need a new test for this" - I'm not going to add these in this round of PRs, and I'd also prefer not to even raise a card to add this in isolation.  Reason being that there's a wider issue of code that lacks unit test coverage which from memory was ~70% according to codecov, so it make more sense to me to raise this as a "very large" card, which starts with identifying how large of an issue it is.  I don't think it's really the time to look at that, though feel free to raise a card yourself if you feel strongly about it.